Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -86,6 +86,12 @@ void MenuBar::AddToolsMenu()
   QMenu* tools_menu = addMenu(tr("Tools"));
   m_wad_install_action = tools_menu->addAction(tr("Install WAD..."), this, SLOT(InstallWAD()));
 
+  // Label will be set by a NANDRefresh later
+  m_boot_sysmenu = tools_menu->addAction(QStringLiteral(""), [this] { emit BootWiiSystemMenu(); });
+  m_boot_sysmenu->setEnabled(false);
+
+  connect(&Settings::Instance(), &Settings::NANDRefresh, [this] { UpdateToolsMenu(false); });
```

Comment:
There's not really a place right now for global signals that aren't related to properties changing, which is why I wondered if there was a place in core code that this could fit... We could probably rename `Settings` to something more generic?